porting justin's edits into GH (#253)
jaybee84/ml-in-rd: content/04.heterogeneity.md

A method commonly used for dimensionality reduction is principal components analysis (PCA).
PCA identifies higher order features, termed _principal components_ (PCs), that are combinations of original features.
The PCs are calculated in a way that maximizes the amount of information (variance) they contain and ensures that each PC is uncorrelated with the other PCs. [@doi:10.1098/rsta.2015.0202]
In practice, researchers often use the first few PCs to reduce the dimensionality without removing what may be important biological variability in the data.
Multidimensional scaling (MDS), t-distributed stochastic neighbor embedding (t-SNE), and uniform manifold approximation and projection (UMAP) are other popular dimension reduction methods, often used for low-dimensional visualization and interpretation of data [@https://arxiv.org/abs/1802.03426; @doi:10.23915/distill.00002]
Testing multiple dimensionality reduction methods may be necessary to obtain a more comprehensive portrait of the data. [@doi:10.1186/s13059-020-02021-3]

> Nguyen and Holmes discuss additional important considerations for using dimensionality reduction methods such as selection criteria and interpretation of results. [@doi:10.1371/journal.pcbi.1006907]

Author: Justin: What does this mean?

Other unsupervised learning approaches such as k-means or hierarchical clustering are used to characterize structure in genomic and imaging data. [@doi:10.1186/1471-2105-9-497; @doi:10.1109/jbhi.2013.2276766]

> 

Reviewer: ```suggestion
```

Since the below paragraph is now two sentences, lets just append them to the previous paragraph.

> Representation learning approaches (which include dimensionality reduction) learn low-dimensional representations (composite features) from the high-dimensional data.

Author: Justin: This paragraph on representation learning doesn’t really say anything different from previous paragraph. How is representational learning different?

Author: ```suggestion
Dimensionality reduction methods are applications of the greater concept of representation learning.
```

Reviewer: ```suggestion
Dimensionality reduction methods are a subset of a type of ML called representation learning.
```

> For example, representation learning through matrix factorization methods can extract features from transcriptomics datasets made of combinations of gene expression values. [@doi:10.1038/s41467-020-14666-6; @doi:10.1093/bioinformatics/btq503; @doi:10.1186/s13059-020-02021-3]

Author: ```suggestion
Various representation learning methods have been used to extract features from transcriptomics datasets made of combinations of gene expression values [@doi:10.1038/s41467-020-14666-6; @doi:10.1093/bioinformatics/btq503; @doi:10.1186/s13059-020-02021-3], predict rare pathologies from images [@doi:10.1016/j.media.2020.101660] (Box 1a) or detect cell populations associated with rare diseases [@doi:10.1038/ncomms14825].
```

Reviewer: ```suggestion
Representation learning methods have been used to extract features from transcriptomics datasets made of combinations of gene expression values [@doi:10.1038/s41467-020-14666-6; @doi:10.1093/bioinformatics/btq503; @doi:10.1186/s13059-020-02021-3], predict rare pathologies from images [@doi:10.1016/j.media.2020.101660] (Box 1a) or detect cell populations associated with rare diseases [@doi:10.1038/ncomms14825].
```

> Representation learning can also be utilized to predict rare pathologies from images [@doi:10.1016/j.media.2020.101660] (Box 1a) or detect cell populations associated with rare diseases [@doi:10.1038/ncomms14825].

Reviewer: ```suggestion
```
